client/components/deposits-overview/deposit-notices.tsx

```diff
@@ -104,22 +104,6 @@ export const NewAccountWaitingPeriodNotice: React.FC = () => (
 	</InlineNotice>
 );
 
-/**
- * Renders a notice informing the user of the number of days it may take for deposits to appear in their bank account.
- */
-export const DepositTransitDaysNotice: React.FC = () => (
-	<InlineNotice
-		icon={ tip }
```

Comment:
Tip isn't used anymore and can be removed from the import.

Reply:
Good catch! Removed in [a14783e](https://github.com/Automattic/woocommerce-payments/pull/9812/commits/a14783e73348204b9f148415a68ce3ab1270a207).